cmd/signal/grpc/server/server.go

```diff
@@ -59,7 +59,18 @@ func (s *server) Signal(stream pb.AVP_SignalServer) error {
 		}
 
 		if payload, ok := in.Payload.(*pb.SignalRequest_Process); ok {
-			s.avp.Process(stream.Context(), payload.Process.Sfu, payload.Process.Pid, payload.Process.Sid, payload.Process.Tid, payload.Process.Eid, payload.Process.Config)
+			if err = s.avp.Process(
+				stream.Context(),
+				payload.Process.Sfu,
+				payload.Process.Pid,
+				payload.Process.Sid,
+				payload.Process.Tid,
+				payload.Process.Eid,
+				payload.Process.Config,
+			); err != nil {
+				log.Errorf("process error: %v", err)
+				return err
```

Comment:
Im not sure if we want to return here, since it will close the stream and prevent future process requests

Reply:
Yep, I will fix it.